UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment/appsettings.json

```diff
@@ -25,17 +19,13 @@
         "NodeName": "Azure",
```

Comment:
Keep entity path blank. This should be configure from library/pipeline.

Reply:
Resolved